backends-velox/src/main/scala/org/apache/gluten/extension/PartialProjectRule.scala

```diff
@@ -27,12 +27,20 @@ case class PartialProjectRule(spark: SparkSession) extends Rule[SparkPlan] {
     plan.transformUp {
       case plan: ProjectExec =>
         val transformer = ColumnarPartialProjectExec.create(plan)
-        if (transformer.doValidate().ok()) {
-          if (transformer.child.asInstanceOf[ColumnarPartialProjectExec].doValidate().ok()) {
+        val validationResult = transformer.doValidate()
+        if (validationResult.ok()) {
+          val childValidationResult =
+            transformer.child.asInstanceOf[ColumnarPartialProjectExec].doValidate()
```

Comment:
Maybe we should respect GlutenConfig `spark.gluten.sql.validation.logLevel`, add some code in https://github.com/apache/incubator-gluten/blob/main/gluten-substrait/src/main/scala/org/apache/gluten/execution/WholeStageTransformer.scala#L86, If the result is not OK, log as this level. I think some refactors may break log, we don't expect catch exception here.

Reply:
OK, then I will roll back. This PR will only simplify the code, and the log will be done later.